spec/integration/_dev/benchmark/system.scenario.spec.yml

```diff
@@ -65,17 +65,12 @@ spec:
           type: object
           additionalProperties: false
           required:
-            - size
+            - total_events
             - template
             - config
             - fields
           properties:
-            size:
-              description: A string describing the amount of data to generate.
-              type: string
-              example: 10MiB
-              deprecated: true # as for https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/94
-            tot_events:
```

Comment:
In order to not break current packages using these variables.
Should we add a json patch to at least add `size` parameter for packages using spec version <3.0.0 ?  cc @jsoriano 

Probably, this would be a similar example:
https://github.com/elastic/package-spec/blob/708b8a7049684091cb699fe840c3eb4ea54e59ba/spec/integration/manifest.spec.yml#L465

Reply:
I haven't found any usage of these `benchmark` folders in the integrations repository.

Commands used in the integrations repository:
```shell
 $ find . -type d -name system  | grep benchmark
 $ find . -type d -name benchmark
```

Maybe it's not needed to add a JSON Patch to re-add the `size` parameter.

WDYT ?

Reply:
@mrodm yes, to my knowledge we added the specs but the reference to them is still only it the specs itself and in the `elastic-package` repo

that's why I was positive to not deal with a migration path like you suggested

once we merge this and make part of v3 https://github.com/elastic/elastic-package/pull/1491 will be green and everything will be fine

what do you think?

Reply:
Yeah, I agree that we don't need the migration path as this is not used yet.

Reply:
> @mrodm yes, to my knowledge we added the specs but the reference to them is still only it the specs itself and in the `elastic-package` repo
> 
> that's why I was positive to not deal with a migration path like you suggested

Perfect, thanks for checking this out. Agreed, it's not need to add the JSON patch :+1: